### DISCUSSION TITLE
Test Results not getting updated on test rail

### DISCUSSION THREAD

**Discussion by rkgour7492** [PeterNgTr/codeceptjs-testrail#94] [General]
I am testing integration of my framework to have test results updated for existing test runs and test plan but unfortunately testrail is not getting updated. It doesnt even show anything debugLog. Just wondering what is wrong. this is my configuration
    testrail: {
      require: 'codeceptjs-testrail',
      host: 'https://xxxxxxx.com',
      user: '<user>',
      password: '<APIkey>',
      suiteId: 1569,
      projectId: 7,
      runId: 13021,
      plan: {
        existingPlanId: 12900
      },
        debugLog: true,
        enabled: true,
        closeTestRun: true
    }

**Comment by kobenguyent** [comment]
Please file an issue with more details if you still encounter the issue with latest version.
